src/test/java/seedu/address/testutil/TaskBuilder.java

```diff
@@ -81,6 +81,14 @@ public TaskBuilder withPhone(String phone) {
         return this;
     }
 
+    /**
+     * Sets the {@code Phone} of the {@code Task} that we are building to be default phone.
```

Comment:
I would recommend making this documentation clearer as well. Add another line to specify that this is only used to construct an empty Phone when the field is not in the task.